catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -229,18 +227,18 @@ public void clear_all() throws ProcessingFault, SystemFault {
   }
 
   @Override
-  public void clear_intervals(int num_intervals)
+  public void clear_intervals(int numIntervals)
```

Comment:
Should rename this method as well assuming this isn't a whitelisted interface

Reply:
Looks like I won't be able to rename these methods since they override interface methods in generated CORBA api code. Would you recommend reverting the parameter name as well?

Reply:
I'd make the parameter names match java standards and just suppress warnings with annotations about the method names.

Reply:
Sonarqube was only concerned with the naming of the parameters. Is there another warning we should be suppressing with the method names?

Reply:
Nope. If the method names don't trigger a a static analysis finding, there's nothing to be suppressed.